plsync/users.py

```diff
@@ -8,4 +8,5 @@
              ('Matt', 'Mathis', 'mattmathis@google.com'),
              ('Peter', 'Boothe', 'pboothe@google.com'),
              ('Fernanda', 'Lavalle', 'lavalle@opentechinstitute.org'),
-             ('Michael', 'Lynch', 'mtlynch@google.com')]
+             ('Michael', 'Lynch', 'mtlynch@google.com'),
```

Comment:
I'll leave him there for now. He still (very generously) lends a hand from time to time. At some point we can remove his account, but that requires more than just removing his name from this file.